app/lib/server/methods/saveSettings.js

```
@@ -41,6 +40,14 @@ Meteor.methods({
 			}
 			settings.updateById(_id, value, editor);
 		});
+		// Throw messages for settings that are not allowed to save!
+		if (settingsNotAllowed.length) {
+			throw new Meteor.Error('error-action-not-allowed', 'Editing settings is not allowed', {
```

This error is not propagated to the user